src/pages/travel.tsx

```diff
@@ -9,9 +14,45 @@ export default function TravelPage() {
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
       </Head>
       <nav className="flex justify-center gap-4">
-        <Link href='/'>Home</Link>
+        <Link href="/">Home</Link>
       </nav>
-      <h2>Travel Page</h2>
+
+      <div className='flex justify-center'>
+        <Image src={wateringPlantsImage} />
```

Comment:
We should probably add an `alt` description for accessibility purposes, even though the image is just decorative

Reply:
added alt message